File: EjerciciosClase/Ejercicio if- else y rangos ,clasificar cliente.md/ClasificacionEdada.java
import java.util.Scanner;
public class ClasificacionEdada {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Ingrese la edad del cliente: ");
        int edad = sc.nextInt();

        if (edad < 18) {
            System.out.println("Cliente Joven");
        } else if (edad <= 64) {
            System.out.println("Cliente Adulto");
        } else {
            System.out.println("Cliente de la Tercera Edad");
        }
    }
}

    

File: EjerciciosClase/Ejercicio if- else y rangos ,clasificar cliente.md/ClasificacionEdada.cpp
#include <iostream>
using namespace std;

int main() {
    int edad;
    cout << "Ingrese la edad del cliente: ";
    cin >> edad;

    if (edad < 18) {
        cout << "Cliente Joven" << endl;
    } else if (edad <= 64) {
        cout << "Cliente Adulto" << endl;
    } else {
        cout << "Cliente de la Tercera Edad" << endl;
    }

    return 0;
}
